src/rewrite_dequantizelinear.cpp

```diff
@@ -0,0 +1,73 @@
+#include <migraphx/rewrite_dequantizelinear.hpp>
+#include <migraphx/instruction.hpp>
+#include <migraphx/iterator_for.hpp>
+#include <migraphx/op/dequantizelinear.hpp>
+#include <migraphx/op/reshape.hpp>
```

Comment:
we can remove all the include "#include <migraphx/op/****.hpp>" and use the make_op function.